tests/src/Common/external/external.depproj

```diff
@@ -5,19 +5,54 @@
     <!-- Given that xunit packages bring with them part of the framework, we need to specify a runtime in order to get the assets
          This RID value doesn't really matter, since the assets we are copying are not RID specific, so defaulting to Windows here
     -->
-    <ProjectJson>$(MSBuildThisFileDirectory)project.json</ProjectJson>
-    <ProjectLockJson>$(MSBuildThisFileDirectory)project.lock.json</ProjectLockJson>
+    <RestoreOutputPath>$(MSBuildThisFileDirectory)obj</RestoreOutputPath>
```

Comment:
It would be a good idea to put this under the output path instead of in the source tree.